### ISSUE TITLE
Issue with ipv6 bind

### ISSUE THREAD

**Issue by guillaumeclair** [haproxytech/dataplaneapi#264]
Hi there,

there is an issue with ipv6 on version 2.5.3, 2.5.4 and 2.6.0. Here is the scenario :

# Creation of backend

curl -X POST --user admin:\<pass\> http://\<host\>:\<port\>/v2/services/haproxy/configuration/backends?version=1112 -d '{"balance": {"algorithm":"roundrobin"},"mode":"http","name":"test_backend"}' -H "Content-type: application/json"
{"balance":{"algorithm":"roundrobin"},"mode":"http","name":"test_backend"}

# Creation of frontend

curl -X POST --user admin:\<pass\> http://\<host\>:\<port\>/v2/services/haproxy/configuration/frontends?version=1113 -H "Content-type: application/json" -d '{"name": "test_frontend", "mode": "http", "default_backend": "test_backend", "maxconn": 2000}'
{"default_backend":"test_backend","maxconn":2000,"mode":"http","name":"test_frontend"}

# Creation of bind

curl -X POST --user admin:\<pass\> "http://\<host\>:\<port\>/v2/services/haproxy/configuration/binds?frontend=test_frontend&version=1114" -d '{"name": "http", "address": "2a01:c9c0:a3:8::3", "port": 80}' -H "Content-type: application/json"
{"name":"http","address":"2a01:c9c0:a3:8::3","port":80}

# Get Bind

curl -X GET --user admin:\<pass\> http://\<host\>:\<port\>/v2/services/haproxy/configuration/binds?frontend=test_frontend
{"_version":1115,"data":[{"name":"http","address":"2a01"}]}

See the address: ipv6 parsing : address is incomplete, and port is not displayed !

Can you please take the issue into account for a further release please ?

thx.

**Comment by gorangalinec** [comment]
I can confirm that this is a bug @mjuraga.

The config line looks correct in the temporary haproxy configuration file 
```
frontend test_frontend
  mode http
  maxconn 2000
  bind 2a01:c9c0:a3:8::3:80 name http
  default_backend test_backend
```

While debugging the client-native code, this is how it appears.

![image](https://user-images.githubusercontent.com/1278300/186907141-144d35be-2e9f-40b6-b6db-9ab7efbe87cb.png)

The IPv6 address used appears to be a valid one according to this validator -> http://sqa.fyicenter.com/1000334_IPv6_Address_Validator.html#Result

**Comment by guillaumeclair** [comment]
Up plz ?

**Comment by mjuraga** [comment]
Hi @guillaumeclair , it is a bug, and we are currently working on fixing it, fix will be included in the following release.

**Comment by mjuraga** [comment]
Hi this should be fixed in `v2.6.1`
